l10n_it_fatturapa_export_zip/views/attachment_view.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_fatturapa_out_attachment_tree">
+        <field name="name">fatturapa.attachment.out.tree.exported</field>
+        <field name="model">fatturapa.attachment.out</field>
+        <field name="inherit_id" ref="l10n_it_fatturapa_out.view_fatturapa_out_attachment_tree"/>
+        <field name="arch" type="xml">
+            <field name="out_invoice_ids" position="after">
+                <field name="zip_exported"/>
+            </field>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_fatturapa_out_attachment_search">
+        <field name="name">fatturapa.attachment.out.search.exported</field>
+        <field name="model">fatturapa.attachment.out</field>
+        <field name="inherit_id" ref="l10n_it_fatturapa_out.view_fatturapa_out_attachment_search"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="zip_exported"/>
```

Comment:
Essendoci già `zip_not_exported` e `zip_already_exported`, `zip_exported` meglio rimuoverlo.

Inoltre, questi filtri sono utili se l'esportazione fosse lanciabile da `fatturapa.attachment.out`/`fatturapa.attachment.in`. Essendo però lanciabile da `account.invoice`, vedo 2 strade:
 - Si mettono questi filtri anche su `account.invoice`
 - Si permette l'esportazione da `fatturapa.attachment.out`/`fatturapa.attachment.in` invece che da `account.invoice`

Reply:
spostato tutto negli attachment